karma.config.js

```diff
@@ -0,0 +1,70 @@
+var argv = require('yargs').argv;
+var path = require('path');
+
+module.exports = function(config) {
+  config.set({
+    browsers: ['PhantomJS'],
+    singleRun: !argv.watch, // just run once by default
+    frameworks: ['mocha', 'chai'],
+    // npm i karma-spec-reporter --save-dev
+    // displays tests in a nice readable format
+    reporters: ['spec'],
+
+    // include some polyfills for babel and phantomjs
+    files: [
+      'node_modules/babel-polyfill/dist/polyfill.js',
+      './node_modules/phantomjs-polyfill/bind-polyfill.js',
+      './test/**/*.js' // specify files to watch for tests
+    ],
+    preprocessors: {
+      // these files we want to be precompiled with webpack
+      // also run tests through sourcemap for easier debugging
+      ['./test/**/*.js']: ['webpack', 'sourcemap']
```

Comment:
I get an error thrown on this line: 
```
03 03 2016 14:53:37.940:ERROR [config]: Invalid config file!
  /Users/arvind/Research/lyra/code/karma.config.js:22
      ['./test/**/*.js']: ['webpack', 'sourcemap']
      ^
SyntaxError: Unexpected token [
```

Should this be valid JSON (`'./test/**/*.js'`)?